src/App.test.js

```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
```

Comment:
Now in app js file we removed 'learn react' text. So, this test case will fail. So, Please remove it.

Reply:
I

Reply:
I have removed that test case as per your suggestion.

Reply:
Resolved